hasher-matcher-actioner/webapp/src/MatchDetails.jsx

```diff
@@ -69,48 +104,71 @@ export default function MatchDetails() {
           </table>
         </Col>
         <Col md={6}>
-          <Image
-            src="https://www.creditabsolute.com/wp-content/uploads/2019/04/block_beach_1.jpg"
-            fluid
-            rounded
-          />
-        </Col>
-      </Row>
-      <Row>
-        <Col md={12}>
-          <table className="table mt-4">
-            <thead>
-              <tr>
-                <th>ID</th>
-                <th>Indicator Type</th>
-                <th>Indicator</th>
-                <th>Created</th>
-                <th>Last Updated</th>
-                <th>Tags</th>
-                <th>Status</th>
-                <th>Partners with Opinions</th>
-                <th>Action Taken</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td>2880742865368386</td>
-                <td>HASH_PDQ</td>
-                <td style={{maxWidth: '250px', overflow: 'hidden'}}>
-                  <span style={{overflow: 'hidden'}} />
-                  acecf3355e3125c8e24e2f30e0d4ec4f8482b878b3c34cdbdf063278db275992
-                </td>
-                <td>31 Jul 2020 6:47pm</td>
-                <td>31 Jul 2020 6:47pm</td>
-                <td>tag1, tag2</td>
-                <td>MALICIOUS</td>
-                <td>app1, app2</td>
-                <td>Delete</td>
-              </tr>
-            </tbody>
-          </table>
+          <BlurUntilHoverImage src={img} />
         </Col>
       </Row>
+      <MatchesList contentKey={id} />
+    </>
+  );
+}
+
+function MatchesList(props) {
+  const [matchDetails, setMatchDetails] = useState(null);
+  // eslint-disable-next-line react/prop-types
+  const {contentKey} = props;
+  useEffect(() => {
+    fetchMatchDetails(contentKey).then(matches => {
+      setMatchDetails(matches.match_details);
+    });
+  }, []);
+
+  return (
+    <>
+      <Spinner hidden={matchDetails !== null} animation="border" role="status">
```

Comment:
ymmv: Wonder if a header here highlighting these as matches could be useful. Maybe just "Matches".

Goal would be to distinguish what is details of the content vs details of the matches. This might just be something we iterate on once we get partner feedback. Absolutely great first pass

Reply:
yeah, we're having a lot of discussions around this page, I agree with you and none of the headings are set in stone right now. Leaving it for now.